release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -86,6 +86,31 @@ For more information, see xref:../installing/installing_vsphere/ipi/installing-r
 You can define tags for compute or control plane machines on an existing cluster by using machine sets.
 For more information, see "Adding tags to machines by using machine sets" for xref:../machine_management/creating_machinesets/creating-machineset-vsphere.adoc#machine-api-vmw-add-tags_creating-machineset-vsphere[compute] or xref:../machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-vsphere.adoc#machine-api-vmw-add-tags_cpmso-config-options-vsphere[control plane] machine sets.
 
+[id="ocp-4.16-postinstallation-configuration_{context}"]
+=== Postinstallation configuration
+
+[id="ocp-4.16-multiarch-tuning-operator_{context}"]
+==== Managing workloads on multi-architecture clusters by using the Multiarch Tuning Operator (Technology Preview)
+
+With this release, you can manage workloads on multi-architecture clusters by using the Multiarch Tuning Operator. This Operator enhances the operational experience within multi-architecture clusters, and single-architecture clusters that are migrating to a multi-architecture compute configuration. It implements the `ClusterPodPlacementConfig` custom resource (CR) to support architecture-aware workload scheduling.
+
+For more information, see xref:../post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.adoc#multiarch-tuning-operator[Managing workloads on multi-architecture clusters by using the Multiarch Tuning Operator].
+
+[IMPORTANT]
+====
+The Multiarch Tuning Operator is a Technology Preview feature only. It does not support clusters with restricted network scenarios.
+====
+
+[id="ocp-4.16-arm-control-plane-with-x86-compute-machines_{context}"]
+==== Support for adding 64-bit x86 compute machines to a cluster with 64-bit ARM control plane machines
+
+This feature provides support for adding 64-bit x86 compute machines to a multi-architecture cluster with 64-bit ARM control plane machines. With this release, you can add 64-bit x86 compute machines to a cluster that uses 64-bit ARM control plane machines and already includes 64-bit ARM compute machines.
```

Comment:
@sr1kar99 
This is true except for these 2 scenarios:

- Baremetal IPI expanding a cluster

https://docs.openshift.com/container-platform/4.15/installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.html#preparing-the-bare-metal-node_ipi-install-expanding

- Baremetal UPI expanding a cluster with the BM Operator: 

https://docs.openshift.com/container-platform/4.15/installing/installing_bare_metal/scaling-a-user-provisioned-cluster-with-the-bare-metal-operator.html

We also don't support adding arm nodes to an x86 cluster as well. I'm not sure if we should mention this limitation here, or in the sections listed? Maybe we address these sections in another PR? What do you think?
Otherwise LGTM

Reply:
Talked to @sr1kar99 offline, will address this in another PR.